### ISSUE TITLE
b.cesm3_cam041_mom.B1850WcMOM.ne30_L58_t061.cice5.cam6.006

### ISSUE THREAD

**Issue by cecilehannay** [NCAR/amwg_dev#124]
**Description:**
Same as #123 with new topo file:
```
bnd_topo = '/glade/p/cgd/amp/pel/topo/files/ne30pg3_gmted2010_bedmachine_nc3000_Laplace0100_20220520.nc'
```
**Case directory:**
Locally (if still available):
/glade/p/cesmdata/cseg/runs/cesm2_0/b.cesm3_cam041_mom.B1850WcMOM.ne30_L58_t061.cice5.cam6.006

On github:
https://github.com/NCAR/amwg_dev/tree/b.cesm3_cam041_mom.B1850WcMOM.ne30_L58_t061.cice5.cam6.006

**Sandbox:**
Locally (if still available):
/glade/work/hannay/cesm_tags/cesm3_cam6_3_041_MOM3

On github:
https://github.com/NCAR/amwg_dev/tree/cesm3_cam6_3_041_MOM3
hash: 5a56ad3

**Diagnostics:**
AMWG diags (if available)
[https://webext.cgd.ucar.edu/BWsc1850MOM/b.cesm3_cam041_mom.B1850WcMOM.ne30_L58_t061.cice5.cam6.006/atm/](https://webext.cgd.ucar.edu/BWsc1850MOM/b.cesm3_cam041_mom.B1850WcMOM.ne30_L58_t061.cice5.cam6.006.atm/)

**Contacts:**
@adamrher, @JulioTBacmeister, @PeterHjortLauritzen, @gustavo-marques

**Comment by gustavo-marques** [comment]
So far (year 11) no issues in the OCN and ICE logs. 
I checked the ocean state for year 10 and the Lab Sea freezes over, but the ice completely melts in the summer. Below is a time series of SST for a point in the middle of the Lab Sea:
![image](https://user-images.githubusercontent.com/11339137/170789350-8ac7b485-6b9a-4ac9-b0ca-72b17e40787b.png)

**Comment by adamrher** [comment]
It looks like 10 years of this run completed. Although the run name does not state it, this is the no-topo-over-ocean run, and should be compared with w/ https://github.com/NCAR/amwg_dev/issues/123. Curious if there are SST differences in the two runs.

**Comment by cecilehannay** [comment]
@adamrher: diags are [here](https://webext.cgd.ucar.edu/BWsc1850MOM/b.cesm3_cam041_mom.B1850WcMOM.ne30_L58_t061.cice5.cam6.006/atm/)

**Comment by adamrher** [comment]
TS changes mirror the elevation changes, around the west coast of the America's, around Greenland. Magnitudes of about 1-2K. Warming over all of Antarctica due to systematic lowering of AIS in the new BedMachine topo dataset?

**Comment by cecilehannay** [comment]
It would be nice to have 20 year to make sure that we capture some of the variability at high latitudes. But the queues are prohibitively slow.

**Comment by JulioTBacmeister** [comment]
Interesting that TS is generally lower in the no-leak TOPO run (006?)

**Comment by adamrher** [comment]
should we resubmit for another 10 years? And the control? We can study the systematic changes around the coastlines with just 10, but I can't tell if differences elsewhere are significant.

**Comment by JulioTBacmeister** [comment]
I think we should do another 10 years.

**Comment by cecilehannay** [comment]
It has been sitting in the queue. I have been talking with Jim about a cheaper pe_layout when the queues are that busy.

**Comment by adamrher** [comment]
agreed, we need a cheaper layout to better balance the queue time vs. throughput. Looking at the timing files, we are supposed to be getting 10 SYPD, but it's taking closer to 10 days to get 10 years through the queue.

**Comment by cecilehannay** [comment]
It is exactly my point. 
I am testing two cheaper pe_layouts right now.
